R/glet.R

```diff
@@ -94,7 +94,7 @@ graphlet_basis <- function(graph, weights = NULL) {
 
   ## Drop all attributes, we don't want to deal with them, TODO
   graph2 <- graph
-  graph2[[9]] <- list(c(1, 0, 1), list(), list(), list())
+  graph2[[igraph_t_idx_attr]] <- list(c(1, 0, 1), list(), list(), list())
```

Comment:
I agree. Yes, please.

I wonder why the tests fail in CI/CD. Can you please try reverting the change to the `.svg` file?

Reply:
It seems slightly better to have all `.Call()` calls in separate R functions. So, this can be a call to a function whose only purpose is to `.Call()` a C function.

Reply:
> I wonder why the tests fail in CI/CD. Can you please try reverting the change to the `.svg` file?

There is one failing tests which is related to `update` old graph. I think it should be fixed after all properties will be fine. For now I just ignored it locally.

Reply:
You can use `skip()` to officially skip the test for now, I had used that in this PR too.